### ISSUE TITLE
Setup of custom component fails

### ISSUE THREAD

**Issue by dcboy87** [pail23/stiebel_eltron_isg_component#15]
First, thank you for creating this component! Unfortunately i can't get it to work. I added your component to HA, restarted and added the ISG integration. After entering all information and submit, this error appears:
![grafik](https://user-images.githubusercontent.com/45015463/224079837-8de92e1d-9fbd-4864-9745-ac65f8c83729.png)

Debuglog is also not very useful:

`2023-03-09 16:51:08.694 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration stiebel_eltron_isg which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-03-09 16:52:36.872 ERROR (MainThread) [custom_components.stiebel_eltron_isg] Error fetching Waermepumpe data: 
2023-03-09 16:52:36.878 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Waermepumpe' for stiebel_eltron_isg integration not ready yet: None; Retrying in background
2023-03-09 16:52:41.995 ERROR (MainThread) [custom_components.stiebel_eltron_isg] Error fetching Waermepumpe data: 
2023-03-09 16:52:52.326 ERROR (MainThread) [custom_components.stiebel_eltron_isg] Error fetching Waermepumpe data: 
2023-03-09 16:53:12.606 ERROR (MainThread) [custom_components.stiebel_eltron_isg] Error fetching Waermepumpe data: 
2023-03-09 16:53:52.684 ERROR (MainThread) [custom_components.stiebel_eltron_isg] Error fetching Waermepumpe data: 
2023-03-09 16:55:13.166 ERROR (MainThread) [custom_components.stiebel_eltron_isg] Error fetching Waermepumpe data: 
2023-03-09 16:56:33.279 ERROR (MainThread) [custom_components.stiebel_eltron_isg] Error fetching Waermepumpe data: 
2023-03-09 16:56:40.741 ERROR (MainThread) [custom_components.stiebel_eltron_isg] Error fetching Waermepumpe data: 
2023-03-09 16:56:40.747 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Waermepumpe' for stiebel_eltron_isg integration not ready yet: None; Retrying in background
`

Do i have to activate something on ISG side?

Versions:
Home Assistant 2023.3.1 
Supervisor 2023.03.1 
Operating System 9.5 
Frontend 20230302.0 - latest

ISG 12.1.0 Build 135
Wärmepumpe WPW-I 12 H 400 Premium

**Comment by xylone** [comment]
No, you don't have to activate anything in the ISG.
Sounds like a connection problem. Did you enter the right IP address? (Or did it change in the meantime)

**Comment by dcboy87** [comment]
Thanks for the answer, but unfortunately the IP is correct and i can access the ISG website (servicewelt). Any other suggestions?

**Comment by xylone** [comment]
Both HA and ISG are on the same subnet? (No DMZ/firewall or whatsoever)
Can you open a terminal on HA and try to ping the ISG ip?

**Comment by dcboy87** [comment]
Yes, everything on the same subnet, ping works, when i try to SSH i get a "connection refused".

**Comment by xylone** [comment]
I think ssh wont be possible to the ISG. 
But otherwise, I'm also confused...
I suppose you already removed and re-added the integration?

**Comment by dcboy87** [comment]
yes, i did it a couple of times. I will wait for a next release, maybe things will get better

**Comment by dcboy87** [comment]
I saw the init.py was changed to get better information on errors, so i gave it a new try:

```
Logger: homeassistant.config_entries
Source: config_entries.py:425
First occurred: 21:20:37 (1 occurrences)
Last logged: 21:20:37

Config entry 'Stiebel Eltron ISG' for stiebel_eltron_isg integration not ready yet: unsupported operand type(s) for /: 'NoneType' and 'int'; Retrying in background
```

But this is also no help for me :-D

**Comment by xylone** [comment]
It almost looks like you have a different version, which have different (unsupported) fields/types.
I think the developer have to jump in ;-)

**Comment by pail23** [comment]
@dcboy87 I implemented a fix in V0.6.0 for this. Could you check if it is working now or if you get a different error message?

**Comment by dcboy87** [comment]
Thank you for helping, now its working.
Its showing the wrong product (WPM 3i instead WPW-I 12 H), but i can see parameters and values 👍

@pail23 i can't find this integration in HACS, will it show up there in the future for easier updating?
